aspnet/Treegrid/Context-Menu.md

```diff
@@ -62,9 +62,9 @@ function customMenuClick(args) {
 
 The following screenshot displays the customization of Context menu in TreeGrid control.
 
-![](Context-Menu_images/Context-Menu_img2.png) 
+![Context menu customization in ASP.NET Web Forms TreeGrid.](Context-Menu_images/Context-Menu_img2.png) 
 
-# Header Context Menu
+### Header Context Menu
```

Comment:
Change it as H2. Here there is under context menu text sub headings only, so it will be H2.

Reply:
changed